samples/CommunityToolkit.Maui.Markup.Sample/Pages/SettingsPage.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using CommunityToolkit.Maui.Behaviors;
+using CommunityToolkit.Maui.Markup.Sample.Constants;
+using CommunityToolkit.Maui.Markup.Sample.Pages.Base;
+using CommunityToolkit.Maui.Markup.Sample.Services;
+using CommunityToolkit.Maui.Markup.Sample.ViewModels;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Layouts;
+
+namespace CommunityToolkit.Maui.Markup.Sample.Pages;
+
+class SettingsPage : BaseContentPage<SettingsViewModel>
+{
+	public SettingsPage(SettingsViewModel settingsViewModel) : base(settingsViewModel, "Settings")
+	{
+		Content = new AbsoluteLayout
+		{
+			Children =
+			{
+				new Label { Text = "Top Stories To Fetch", TextColor = ColorConstants.PrimaryTextColor, HorizontalTextAlignment = TextAlignment.Center }
+					.LayoutFlags(AbsoluteLayoutFlags.XProportional | AbsoluteLayoutFlags.WidthProportional)
+					.LayoutBounds(0,0,1,40),
+
+				new Entry { Keyboard = Keyboard.Numeric, BackgroundColor = Colors.White }
+					.LayoutFlags(AbsoluteLayoutFlags.XProportional | AbsoluteLayoutFlags.WidthProportional)
+					.LayoutBounds(0.5,45,0.8,40)
+					.Behaviors(new NumericValidationBehavior
+					{
+						MinimumValue = SettingsService.MinimumStoriesToFetch,
+						MaximumValue = SettingsService.MaximumStoriesToFetch,
+						Flags = ValidationFlags.ValidateOnValueChanged,
+						InvalidStyle = new Style<Entry>((Entry.TextColorProperty, Colors.Red)),
+						ValidStyle = new Style<Entry>((Entry.TextColorProperty, ColorConstants.PrimaryTextColor)),
+					})
+					.Bind(Entry.TextProperty, nameof(SettingsViewModel.NumberOfTopStoriesToFetch)),
+
+				new Label { TextColor = ColorConstants.PrimaryTextColor, HorizontalTextAlignment = TextAlignment.Center }

```

Comment:
We can remove `HorizontalTextAlignment = TextAlignment.Center` now that we've added `.TextCenter()` below
```suggestion
				new Label { TextColor = ColorConstants.PrimaryTextColor }

```